### PR TITLE
Fixing issue with post-build analysis for quarkus

### PR DESCRIPTION
**Pull request by harshmadhani**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java [hunk 1]
```diff
@@ -48,13 +48,13 @@ private static void postBuildCheck(String stagingPath, String productName) {
...
 
-            String latest_build = quarkusLatestBuild.select("a[href]").text().replace("/", "");
-            String old_build = quarkusLatestBuild.nextElementSibling().select("a[href]").text().replace("/", "");
+            String latest_build = quarkusBuilds.first().select("a[href]").text().replace("/", "");
+            String old_build = quarkusBuilds.get(1).select("a[href]").text().replace("/", "");
 
-            log.info("Latest build is {}", latest_build);
-            log.info("Old build is {}", old_build);
+            log.info("Latest build is " + latest_build);
+            log.info("Old build is " + old_build);
```
Comment (rnc) [inline pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java:57]: What is the reason for changing the logging format please?

Comment (harshmadhani) [reply pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java:57]: None, mistake on my side :) . Will revert it back to the original one

**Comment by thescouser89** [comment]
Hi! Please run a ```mvn clean install``` beforehand so that the maven formatter is applied correctly.

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java [hunk 2]
```diff
@@ -70,8 +70,8 @@ private static void postBuildCheck(String stagingPath, String productName) {
                     + CollectionUtils.subtract(newDependencies, oldDependencies);
             String oldBuildInfo = "Community Dependencies present in old build which are not present in new build are "
                     + CollectionUtils.subtract(oldDependencies, newDependencies);
-            log.info("Build info for new build is {}", newBuildInfo);
-            log.info("Build info for old build is {}", oldBuildInfo);
+            log.info("Build info for new build is " + newBuildInfo);
+            log.info("Build info for old build is " + oldBuildInfo);
```
Comment (rnc) [inline pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java:74]: Still got logging changes.... ?

**Comment by rnc** [comment]
Can you also squash the commits please?

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java (hunk 2 above)
Comment (harshmadhani) [reply pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java:74]: I will raise a new PR with squashed commits and formatting changes. Running mvn clean install didn't work too :( Need to find some other way I think

**Pull request closed without merging**